server/chat-plugins/mafia.ts

```diff
@@ -330,9 +346,12 @@ class Mafia extends Rooms.RoomGame<MafiaPlayer> {
 		this.started = false;
 
 		this.theme = null;
+		this.hostid = toID("");
+		this.host = "";
 
 		this.hostid = host.id;
-		this.host = Utils.escapeHTML(host.name);
+		this.host = Utils.escapeHTML(host.name) && '';
```

Comment:
```suggestion
		this.host = Utils.escapeHTML(host.name);
```
`&& '';` will mean this always returns `''`, you probably meant `|| '';` to say "use the HTML escaped host name or an empty string as a default". However, that isn't needed, if you look at the definition for `escapeHTML`, you can see that it always returns a string (even an empty one if needed).